<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Win_login</class>
    <widget class="QWidget" name="Win_login">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>500</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>800</width>
                <height>500</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>800</width>
                <height>500</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Win_Login</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                        <widget class="QLabel" name="lable_login">
                            <property name="font">
                                <font>
                                    <family>黑体</family>
                                    <pointsize>30</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>某大学贫困生资助系统</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                                <spacer name="horizontalSpacer_4">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QLabel" name="login_img">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="pixmap">
                                        <pixmap>img/login.jpg</pixmap>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QVBoxLayout" name="verticalLayout">
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout">
                                            <item>
                                                <spacer name="horizontalSpacer_2">
                                                    <property name="orientation">
                                                        <enum>Qt::Horizontal</enum>
                                                    </property>
                                                    <property name="sizeHint" stdset="0">
                                                        <size>
                                                            <width>40</width>
                                                            <height>20</height>
                                                        </size>
                                                    </property>
                                                </spacer>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label_2">
                                                    <property name="font">
                                                        <font>
                                                            <family>黑体</family>
                                                        </font>
                                                    </property>
                                                    <property name="text">
                                                        <string>用户</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLineEdit" name="edit_username"/>
                                            </item>
                                            <item>
                                                <spacer name="horizontalSpacer">
                                                    <property name="orientation">
                                                        <enum>Qt::Horizontal</enum>
                                                    </property>
                                                    <property name="sizeHint" stdset="0">
                                                        <size>
                                                            <width>40</width>
                                                            <height>20</height>
                                                        </size>
                                                    </property>
                                                </spacer>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                                            <item>
                                                <spacer name="horizontalSpacer_5">
                                                    <property name="orientation">
                                                        <enum>Qt::Horizontal</enum>
                                                    </property>
                                                    <property name="sizeHint" stdset="0">
                                                        <size>
                                                            <width>40</width>
                                                            <height>20</height>
                                                        </size>
                                                    </property>
                                                </spacer>
                                            </item>
                                            <item>
                                                <widget class="QLabel" name="label">
                                                    <property name="font">
                                                        <font>
                                                            <family>黑体</family>
                                                        </font>
                                                    </property>
                                                    <property name="text">
                                                        <string>密码</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QLineEdit" name="edit_password">
                                                    <property name="echoMode">
                                                        <enum>QLineEdit::Password</enum>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <spacer name="horizontalSpacer_6">
                                                    <property name="orientation">
                                                        <enum>Qt::Horizontal</enum>
                                                    </property>
                                                    <property name="sizeHint" stdset="0">
                                                        <size>
                                                            <width>40</width>
                                                            <height>20</height>
                                                        </size>
                                                    </property>
                                                </spacer>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <widget class="QComboBox" name="combox_login">
                                            <property name="font">
                                                <font>
                                                    <family>黑体</family>
                                                </font>
                                            </property>
                                            <item>
                                                <property name="text">
                                                    <string>学生</string>
                                                </property>
                                            </item>
                                            <item>
                                                <property name="text">
                                                    <string>老师</string>
                                                </property>
                                            </item>
                                        </widget>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                                            <item>
                                                <widget class="QPushButton" name="button_login">
                                                    <property name="font">
                                                        <font>
                                                            <family>黑体</family>
                                                        </font>
                                                    </property>
                                                    <property name="text">
                                                        <string>登录</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <spacer name="horizontalSpacer_7">
                                                    <property name="orientation">
                                                        <enum>Qt::Horizontal</enum>
                                                    </property>
                                                    <property name="sizeType">
                                                        <enum>QSizePolicy::Fixed</enum>
                                                    </property>
                                                    <property name="sizeHint" stdset="0">
                                                        <size>
                                                            <width>40</width>
                                                            <height>20</height>
                                                        </size>
                                                    </property>
                                                </spacer>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="button_register">
                                                    <property name="font">
                                                        <font>
                                                            <family>黑体</family>
                                                        </font>
                                                    </property>
                                                    <property name="text">
                                                        <string>注册</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_3">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
